daml-lf/kv-transaction-support/src/main/scala/com/daml/lf/kv/TransactionNormalizer.scala

```diff
@@ -6,12 +6,12 @@ package com.daml.lf.kv
 import com.daml.lf.transaction._
 import com.daml.lf.value.Value.ContractId
 
-object Normalizer {
```

Comment:
Shall I move this to the other normalizers and get rid of this package (`kv-transaction-support`)?

Reply:
I suggest not to, because this normalizer is a runtime one and not an integrity checker one. If I remember correctly, it originated from a past effort of runtime storage reduction.

Reply:
We moved this logic there recently to make it clear that this module is maintained by the Daml Runtime team.